homeassistant/components/websocket_api/http.py

```diff
@@ -71,7 +71,8 @@ def __init__(self, hass: HomeAssistant, request: web.Request) -> None:
         self.hass = hass
         self.request = request
         self.wsock = web.WebSocketResponse(heartbeat=55)
-        self._to_write: asyncio.Queue = asyncio.Queue(maxsize=MAX_PENDING_MSG)
+        self._message_queue: deque = deque()
```

Comment:
Please add a comment explaining why we're using a pair of a `deque` + `asyncio.Future` instead of an `asyncio.Queue`

Reply:
924cd6250c2c5f7f0e413d47fc1d174eb60a32db